src/components/Select.js

```diff
@@ -66,16 +67,25 @@ const Select = React.createClass({
     if (!isMobile) {
       this.setState({
         isOpen: !this.state.isOpen
+      }, () => {
+        if (document.getElementById('items')) {
+          const items = document.getElementById('items');
+
+          items.scrollTop = this.state.scrollLocation;
+        }
```

Comment:
I think what is happening is that when you call `setScrollPosition`, the DOM is already done updating. As soon as the `state` of your component changes, `render` is called to update the component.